pkg/services/system_intakes.go

```diff
@@ -284,7 +284,7 @@ func NewSubmitSystemIntake(
 		// only send an email when everything went ok
 		err = emailReviewer(incoming.Requester, incoming.ID)
 		if err != nil {
-			return &models.SystemIntake{}, err
+			appcontext.ZLogger(ctx).Error("Submit Intake email failed to send: ", zap.Error(err))
```

Comment:
Non-blocking question: is this enough info for the Operators to do anything about this issue? Maybe adding the ID to the log line would help know how to remediate the problem if it's just some transient problem in the email sending layer.